### ISSUE TITLE
Minification error

### ISSUE THREAD

**Issue by fkling** [fkling/astexplorer#101]
Webpack (therefore UglifyJS) has problems minifying some files:

```
ERROR in app-6fe76ad4fb1070d5cdee.js from UglifyJs
Unexpected token punc «,», expected arrow «=>» [./~/codemirror/mode/javascript/javascript.js:363,0]

ERROR in 1-595b1745447d2e8d1d87.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/codemirror/addon/tern/tern.js:324,0]

ERROR in 15-97b6ed0349ff4e6be2c6.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/recast/lib/printer.js:196,0]

ERROR in 16-9ab0ec6c3bb5ef480ff7.js from UglifyJs
Unexpected token punc «)», expected arrow «=>» [./~/shift-parser/dist/pattern-acceptor.js:146,0]

ERROR in 18-4a8de760966ccf94578c.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/babel-generator/lib/printer.js:79,0]

ERROR in 19-80325d20605b9cfcf9bf.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/recast/lib/printer.js:196,0]

ERROR in 20-dc214efd0de9edbdb9bd.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/typescript/lib/typescript.js:30938,0]

ERROR in 22-2661016f44c06a5eb7c6.js from UglifyJs
Unexpected token punc «)», expected arrow «=>» [./~/webidl2/lib/webidl2.js:234,0]

ERROR in 23-5b2dcb9b24f8a5691072.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/escodegen/escodegen.js:907,0]

ERROR in 26-6d6700619899083ed12c.js from UglifyJs
Unexpected token punc «;», expected arrow «=>» [./~/esprima/esprima.js:2925,0]
```

**Comment by fkling** [comment]
It seems webpack is using the the uglify-js version what we are including as parser (the harmony branch version).

**Comment by fkling** [comment]
I don't seem to be able to make webpack use its own uglify-js version. What do you think @RReverser , should we just use the publish uglify-js version instead of the harmony branch one? This makes webpack work properly.

**Comment by RReverser** [comment]
Doesn't shrinkwrap solve this kind of problems? (IIUC, it should)
I believe having #harmony as UI parser is nice for seeing future nodes.

**Comment by fkling** [comment]
Same problem. In fact, the shrinkwrap file only includes a single uglify JS version, 2.6.1, from the harmony branch. Since this version satisfies webpacks requirements, I guess webpack doesn't get its own copy...

**Comment by fkling** [comment]
Kind of fixed here: 790d032a8a428b4aae7f07e3d7c5c4ce08b11492 .
